wandb/sdk/internal/meta.py

```diff
@@ -214,8 +214,6 @@ def probe(self):
         if not self._settings.disable_code:
             if self._settings.program_relpath is not None:
                 self.data["codePath"] = self._settings.program_relpath
-            else:
```

Comment:
Did you want to get rid of this `else` too? I think it changes the behavior. Previously, everything below would only run if self_settings.program_relpath is None. Looking at what's intended here, we probably want to do the jupyter stuff regardless of what's going on with relpath (though it's probably true that we never have relpath in the jupyter case).

Since you're in here, would you mind refactoring a bit to remove some levels of nesting? Guard clauses would help.

Reply:
oh wait yeah the else is a mistake, good catch